src/tests/ftest/util/general_utils.py

```diff
@@ -605,17 +605,25 @@ def stop_processes(hosts, pattern, verbose=True, timeout=60):
     result = {}
     log = getLogger()
     log.info("Killing any processes on %s that match: %s", hosts, pattern)
+
+    if added_filter:
+        ps_cmd = "ps x | grep -E {} | grep -vE {}".format(pattern, added_filter)
+    else:
+        pc_cmd = "pgrep --list-full {}".format(pattern)
```

Comment:
Looks like I had a typo in my suggestion. My apologies.
```suggestion
        ps_cmd = "pgrep --list-full {}".format(pattern)
```

Reply:
1b6386dc60654a50bf68dddacfb23452625090e0